dpnp/random/dpnp_algo_random.pyx

```diff
@@ -113,6 +114,11 @@ ctypedef void(*fptr_dpnp_rng_pareto_c_1out_t)(void * , const double, const size_
 ctypedef void(*fptr_dpnp_rng_poisson_c_1out_t)(void * , const double, const size_t) except +
 ctypedef void(*fptr_dpnp_rng_power_c_1out_t)(void * , const double, const size_t) except +
 ctypedef void(*fptr_dpnp_rng_rayleigh_c_1out_t)(void * , const double, const size_t) except +
+ctypedef void(*fptr_dpnp_rng_shuffle_c_1out_t)(void * ,
+                                               const size_t,
+                                               const size_t,
+                                               const size_t,
+                                               const size_t) except +
 ctypedef void(*fptr_dpnp_rng_srand_c_1out_t)(const size_t) except +
```

Comment:
I'm not sure about it but as far as I remember we have common place for all such `typedef`. Somewhere in algo.pxd.
perhaps it is a good idea to collect all function type definitions into one place. In a separate file `algo_c_types.pxd`
may be it is bad idea. sorry.

Reply:
I don't understand why is this needed. This typedefs is not for public cython interface. Why is it needed on included .pxd
We need to think carefully about the necessity of these changes.